README.md

```diff
@@ -0,0 +1,209 @@
+// }
+
+email.Headers.Cc
+// []mail.Address{
+//  {Name: "Dan Recipient", Address: "dan.recipient@example.com"}, 
+//  {Name: "Eve Recipient", Address: "eve.recipient@example.com"},
+// }
+
+email.Headers.Bcc
+// []mail.Address{
+//  {Name: "Frank Recipient", Address: "frank.recipient@example.com"}, 
+//  {Name: "Grace Recipient", Address: "grace.recipient@example.com"},
+// }
+```
+
+get custom headers:
+
+```go
+email.Headers.ExtraHeaders
+// map[string][]string{
+//    "X-Clacks-Overhead": {"GNU Terry Pratchett"},
+// }
+```
+
+get decoded bodies:
+
+```go
+email.Text
+// "The quick brown fox jumps over a lazy dog..."
+
+email.HTML
+// "<html><div dir="ltr"><p>The quick brown fox jumps over a lazy dog..."
+```
+
+inline files:
+
+```go
+email.InlineFiles
+// []InlineFile{
+//    {
+//        ContentType: ContentTypeHeader{
+//            ContentType: "image/jpeg",
+//            Params: map[string]string{
+//                "name": "inline-jpg-image-without-disposition.jpg",
+//            },
+//        },
+//        ContentDisposition: ContentDispositionHeader{
+//            ContentDisposition: "",
+//            Params:             map[string]string(nil),
+//        },
+//        Data: []byte{255, ...},
+//    },
+//    {
+//        ContentID: "inline-jpg-image.jpg@example.com",
+//        ContentType: ContentTypeHeader{
+//            ContentType: "image/jpeg",
+//            Params: map[string]string{
+//                "name": "inline-jpg-image-name.jpg",
+//            },
+//        },
+//        ContentDisposition: ContentDispositionHeader{
+//            ContentDisposition: inline,
+//            Params: map[string]string{
+//                "filename": "inline-jpg-image-filename.jpg",
+//            },
+//        },
+//        Data: []byte{255, ...},
+//    },
+// }
+```
+
+and attached files:
+
+```go
+email.AttachedFiles
+// []AttachedFile{
+//    {
+//        ContentType: ContentTypeHeader{
+//            ContentType: "application/pdf",
+//            Params: map[string]string{
+//                "name": "attached-pdf-name.pdf",
+//            },
+//        },
+//        ContentDisposition: ContentDispositionHeader{
+//            ContentDisposition: attachment,
+//            Params: map[string]string{
+//                "filename": "attached-pdf-filename.pdf",
+//            },
+//        },
+//        Data: []byte{37, ...},
+//    },
+//    {
+//        ContentType: ContentTypeHeader{
+//            ContentType: "application/pdf",
+//            Params: map[string]string{
+//                "name": "attached-pdf-without-disposition.pdf",
+//            },
+//        },
+//        ContentDisposition: ContentDispositionHeader{
+//            ContentDisposition: "",
+//            Params:             map[string]string(nil),
+//        },
+//        Data: []byte{37, ...},
+// }
+```
+
+The same parser and methods will work for other languages, text encodings, 
+and transfer encodings:
```

Comment:
Super nit-picking, but should it be written like this?

```suggestion
and transfer-encodings:
```

Same for `(...) Quoted-Printable Content Transfer Encodings` a little bit lower down in this document should probably be `(...) Quoted-Printable Content-Transfer-Encodings`.

Reply:
Fixed in https://github.com/mnako/letters/pull/13/commits/a6a447526d281a055386a1e37de68c6485843ffb, thank you 🙇

Reply:
I have also added you to CONTRIBUTORS.md in https://github.com/mnako/letters/pull/13/commits/a597b11d107089c9d1962b131a061ac741e51955.